cmd/todo/main.go

```diff
@@ -49,13 +49,14 @@ func main() {
 		}
 		todos.Print()
 	case *complete > 0:
-		err := todos.Complete(*complete)
+		err := todos.Complete(*complete, historyFile)
```

Comment:
This historyFile cannot be pushed into the Complete function like this, here store function should be used to store completed todos in the history file